debian/rules

```diff
@@ -1,18 +1,50 @@
 #!/usr/bin/make -f
+#
+# Build Debian package using https://github.com/spotify/dh-virtualenv
+#
+# The below targets create a clean copy of the workdir via
```

Comment:
I'm afraid I'm completely failing to grok this. why does the presence of the `debian` dir make it go haywire?

(`pip install .` works just fine for me)

Reply:
I'll remove it, this was from the cookiecutter, it looks like modern pip is more sensible.

Reply:
\o/